packages/odyssey-react-mui/src/OdysseyTranslationProvider.tsx

```diff
@@ -12,38 +12,41 @@
 
 import { ReactNode, useEffect } from "react";
 
-import { SupportedLanguages } from "./OdysseyTranslationProvider.types";
+import { OktaSupportedLanguages } from "./OdysseyTranslationProvider.types";
 
 import { i18n, defaultNS, resources } from "./i18n";
 import { I18nextProvider } from "react-i18next";
 
-export type TranslationOverrides = {
-  [key in SupportedLanguages]?: Partial<(typeof resources)["en"]>;
-};
+export type TranslationOverrides = Record<
+  string,
```

Comment:
Is this mainly to allow any keys that will be defined in a locale bundle that is not in the supported list ?

Reply:
no this type is here so that you can choose to provide a subset of keys.

Otherwise if we use the typing from the `i18n.ts` file, we will need to provide ALL keys